pull_request_template.md

```diff
@@ -0,0 +1,27 @@
+## Description 📝
+
+Please provide a brief description of the purpose and goals of this pull request.
+
+## Changes 💬
+
+- High-level overview of the changes introduced.
+- Step-by-step breakdown of the implementation, if applicable.
+
+## Checklist✅
```

Comment:
Please leave a space between emoji and text like you have added in the previous heading

Reply:
How is this ?? 👇 , any further changes you want me to make ?
![Screenshot 2023-05-29 at 12-12-13 Editing GyanaGuru_pull_request_template md at pull_request_template · DSR1729_GyanaGuru · GitHub](https://github.com/PiyushKalyanpy/GyanaGuru/assets/115932772/1b801757-1034-46e9-b516-2707381af9ca)